client/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import {combineReducers} from 'redux'
+
+import orders from './orders'
+
+export default combineReducers({
+  orders
```

Comment:
I think `current-order` is a better name. We will probably have an `orders` reducer when we implement history.